src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -93,114 +87,112 @@ public ServiceBusHelper serviceBusHelper() {
     }
 
     @Scheduled(fixedDelayString = "${scheduling.task.scan.delay}")
-    public void processBlobs() throws IOException, StorageException, URISyntaxException {
-        for (CloudBlobContainer container : cloudBlobClient.listContainers()) {
+    public void processBlobs() throws IOException {
+        List<ContainerItem> containerItems = azureStorageHelper.listContainers().blockingGet().body().containerItems()
+            .stream()
+            .filter(c -> !c.name().equals("$web"))
+            .collect(toList());
+
+        log.info("Processing blobs in containers: {}", containerItems.stream()
+            .map(ContainerItem::name)
+            .collect(toList()));
+
+        for (ContainerItem container : containerItems) {
             processZipFiles(container);
         }
     }
 
-    private void processZipFiles(CloudBlobContainer container)
-        throws IOException, StorageException, URISyntaxException {
-        log.info("Processing blobs for container {}", container.getName());
+    private void processZipFiles(ContainerItem container)
```

Comment:
I hope we are going to refactor this to actually use the RXJava properly?
Im guessing refactoring this might be to much for one card ?